boards/samr21-xpro/Makefile.dep

```diff
@@ -3,6 +3,11 @@ ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
   USEMODULE += gnrc_nomac
 endif
 
+ifneq (,$(filter periph_hwrng,$(FEATURES_REQUIRED)))
+  USEMODULE += at86rf233
+  USEMODULE += gnrc
```

Comment:
I had to include this, because the at86rfxx driver is not included by default in the samr21-xpro Makefile, but it is neccesary for the hwrng implementation. And since the FEATURES_PROVIDED clauses are only available for boards and not for drivers, I found this to be the easiest solution to provide the dependencies without burdening the boards with at86rf2xx and gnrc all the time.